lms/djangoapps/certificates/views.py

```diff
@@ -481,9 +468,27 @@ def render_html_view(request, user_id, course_id):
     configuration = CertificateHtmlViewConfiguration.get_config()
     context.update(configuration.get('default', {}))
 
+    # Translators:  'All rights reserved' is a legal term used in copyrighting to protect published content
+    reserved = _("All rights reserved")
+    context['copyright_text'] = _('&copy; {year} {platform_name}. {reserved}.').format(
```

Comment:
For the record I strongly prefer leaving &copy; and &amp; out of translation strings. If you absolutely must put the symbols in, better to put the literal symbol in and then escape the string once the call to _() returns.

It's not good if our strings can't handle literal fancy characters - translators may add them, we have no control over that.